docs/gcp/osd_preexisting_vpc.md

```diff
@@ -0,0 +1,168 @@
+# Creating a OSD in GCP with Existing VPCs
+
+**Roberto Carratalá, Andrea Bozzoni**
+
+*Last updated 07/06/2022*
+
+> **Tip** The official documentation for installing a OSD cluster in GCP can be found [here](https://docs.openshift.com/dedicated/osd_cluster_create/creating-a-gcp-cluster.html).
+
+For deploy an OSD cluster in GCP using existing Virtual Private Cloud (VPC) you need to implement some prerequisites that you must create before starting the OpenShift Dedicated installation though the OCM:
+
+## Prerequisites
+
+* [gcloud CLI](https://cloud.google.com/sdk/gcloud)
+* [jq](https://stedolan.github.io/jq/download/)
+
+NOTE: Also the GCloud Shell can be used, and have the gcloud cli among other tools preinstalled.
+
+## Option 1 - Generate OSD VPC and Subnets using GCloud CLI
+
+As mentioned before, for deploy OSD in GCP using existing GCP VPC, you need to provide and create beforehand a GCP VPC and two subnets (one for the masters and another for the workers nodes).
+
+1. Login and configure the proper GCP project where the OSD will be deployed:
+
+    ```sh
+    export PROJECT_NAME=<google project name>
+    gcloud auth list
+    gcloud config set project $PROJECT_NAME
+    gcloud config list project
+    ```
+
+NOTE: this commands can be executed within the 
+
+2. Export the names of the vpc and subnets:
+
+    ```sh
+    export REGION=<region name>
+    export OSD_VPC=<vpc name>
+    export MASTER_SUBNET=<master subnet name>
+    export WORKER_SUBNET=<worker subnet name>
+    ```
+
+3. Create a custom mode VPC network:
+
+    ```sh
+    gcloud compute networks create $OSD_VPC --subnet-mode=custom
+    gcloud compute networks describe $OSD_VPC
+    ```
+
+NOTE: we need to create the mode custom for the VPC network, because the auto mode generates automatically the subnets with IPv4 ranges with [predetermined set of ranges](https://cloud.google.com/vpc/docs/subnets#ip-ranges).
+
+4. This example is using the standard configuration for these two subnets:
+
+    ```md
+    master-subnet - CIDR 10.0.0.0/17   - Gateway 10.0.0.1
+    worker-subnet - CIDR 10.0.128.0/17 - Gateway 10.0.128.1
+    ```
+
+5. Create the GCP Subnets for the masters and workers within the previous GCP VPC network:
+
+    ```sh
+    gcloud compute networks subnets create $MASTER_SUBNET --network=$OSD_VPC --range=10.0.0.0/17 --region=$REGION
```

Comment:
wrap the commands with `\` so they fit easier on narrow screens.  I would make this at least two lines.

Reply:
ack. I'll adjust all the code blocks then